lightstep/resource_stream_test.go

```diff
@@ -114,6 +116,149 @@ resource "lightstep_stream" "import-stream"{
 	})
 }
 
+func TestAccDeleteStreamWithDependentResources(t *testing.T) {
+	streamConfig := `
```

Comment:
nit/personal preference: if there's a streamConfig2 I often appreciate when the first is numbered

Reply:
done!